scripts/ytdl

```diff
@@ -87,7 +88,18 @@ def download(video, audio=False, stream=None, preftype=None):
 
     xprint(dl_str.format(stream.filename, size))
     xprint("-Quality: %s; Format: %s" % (stream.quality, stream.extension))
-    stream.download(quiet=False)
+
+    if outfilepath is None: outfilepath = stream.title
+
+    outdirpath = os.path.dirname(outfilepath)
+    outfilename = os.path.basename(outfilepath)
+    validChars = set('-_. ' + string.ascii_letters)
```

Comment:
I think this is too restricted; non-acsii characters can be valid filenames (imagine downloading something with a name in Chinese/Russian/etc.). And this eliminates even numbers.